src/backend/package-lock.json

```diff
@@ -3391,6 +3391,11 @@
         "node": "*"
       }
     },
+    "node_modules/buffer-equal-constant-time": {
```

Comment:
Did we update a package as part of this change? If so, there should be a package.json change as well.

And if not, we should not update "package-lock" without any updates.